src/GSoft.ComponentModel.DataAnnotations/UrlOfKindAttribute.cs

```diff
@@ -18,7 +18,6 @@ public UrlOfKindAttribute(UriKind kind)
         this.Kind = kind;
     }
 
-    [Url]
```

Comment:
Clearly a mistake I forgot to erase back then, has no impact on the behavior of the lib